maltd-frontend/src/components/hoc/AuthenticationGuard.js

```diff
@@ -0,0 +1,46 @@
+import React, { useState, useEffect } from "react";
+import Keycloak from "keycloak-js";
+import MainPage from "../page/MainPage/MainPage";
+
+const KEYCLOAK = {
+  url: "https://sso-dev.pathfinder.gov.bc.ca/auth",
+  realm: "ezb8kej4",
```

Comment:
All of these configurations need to be environment variables as well. Mainly the url. This code will only work in Dev but not Test and Production :)